### ISSUE TITLE
The People Want: Comments

### ISSUE THREAD

**Issue by oleoneto** [alvarosness/django_project#5]
So, remember your posts? I want to comment on those. Have some burning comments waiting to light up a discussion. Please act in compliance with my request as per our [friendship](https://en.wikipedia.org/wiki/Friendship) agreement.

**Comment by alvarosness** [comment]
Yes! I do remember them as clearly as if it was yesterday. Will direct the request to my supervisor for approval.
